pyproject.toml

```
@@ -145,3 +147,10 @@ max-complexity = 10
     "F401", # unused import
     "F403", # star imports
 ]
+
+[tool.pytest.ini_options]
+addopts = "--strict-markers"
```

```suggestion
addopts = "-m not 'agent-jobs'"
strict-markers = true
```
This will prevent the agent test from running unless the marker is explicitly passed.

---

BTW, your original use of strict markers was correct.
Looks like you can only use it as a CLI or `addopts` 
https://docs.pytest.org/en/stable/reference/reference.html#confval-markers